doc/api/esm.md

```diff
@@ -431,6 +431,46 @@ thrown:
 }
 ```
 
+#### Self-referencing a package using its name
+
+Self-referencing a package's name in an `import` is allowed. For example,
```

Comment:
```suggestion
Within a package, the values defined in the package’s `package.json` `"exports"` field can be referenced via the package’s name. For example,
```

(and fix line length)

Reply:
Not sure about this change. It trades off simplicity for precision. Not sure the tradeoff is worth it in this case.

I will, of course, defer to your judgement.

Reply:
I was confused that this only mentions `import`, when the feature applies to both `require` and `import`. I know you mention `require` further down, but I think we should lead off with that this feature is about `"exports"`, not `import`—as in, it works only when a package has `"exports"`, not when code uses `import`.

Reply:
Fixed

Reply:
Ah. Good point!